config/site_settings.yml

```diff
@@ -2403,7 +2403,7 @@ uncategorized:
 
   use_polymorphic_bookmarks:
     client: true
-    default: false
+    default: true
```

Comment:
This is very interesting. It is probably fine, but it looks like `use_polymorphic_bookmarks` does not have any effect even if it true or false, so that means it could be removed?

Reply:
It still affects plugins -- I will need to make PRs for encrypt, assign, kolide to remove their checks for `use_polymorphic_bookmarks` after this one is merged.